test/support/api/patient.js

```diff
@@ -14,11 +14,11 @@ Cypress.Commands.add('routePatient', (mutator = _.identity) => {
       let included = [];
 
       included = getIncluded(included, this.fxGroups, 'groups');
-      included = getIncluded(included, this.fxPatientFields, 'patient-fields');
+      included = getIncluded(included, this.fxFields, 'fields');
```

Comment:
@paulfalgout should these be getting pushed onto the included array?

Reply:
I'm dumb and didn't see that `included` is being passed to `getIncluded`. carry on.